package.json

```diff
@@ -151,14 +150,14 @@
   },
   "packageManager": "yarn@3.8.1",
   "resolutions": {
-    "axios": "^1.6.5",
+    "axios": "^1.6.8",
     "cookiejar": "^2.1.4",
     "express": "^4.19.2",
     "follow-redirects": "^1.15.6",
     "http-cache-semantics": "^4.1.1",
-    "qs": "^6.11.2",
-    "puppeteer": "^22.0.0",
+    "qs": "^6.12.1",
+    "puppeteer": "^22.7.1",
     "semver": "^7.6.0",
-    "tar": "7.0.0"
+    "tar": "7.0.1"
```

Comment:
Do you actually need these resolutions? What happens if you take them all out? I've tried locally and audit is not reporting any issues..

Reply:
il take a look